cypress/support/validations/decodeValidation.js

```diff
@@ -130,10 +130,9 @@ class decodeValidations {
 
         if (decode.includes(param)) {
           const indexOfParam = decode.indexOf(param);
-          const extractedData = decode.slice(
-            indexOfParam + param.length + 2,
-            decode.indexOf('</', indexOfParam + 1)
```

Comment:
will this fail rajankikas changes ? If this was for advertising fix, we have another ticket